src/test/java/org/codeforamerica/shiba/output/caf/SnapExpeditedEligibilityDeciderTest.java

```diff
@@ -33,7 +34,7 @@
     @BeforeEach
     void setup() {
         applicationData.setPagesData(pagesData);
-        when(mockUtilityDeductionCalculator.calculate(any())).thenReturn(0);
+        when(mockUtilityDeductionCalculator.calculate(any())).thenReturn(new Money(0));
```

Comment:
Seems like new Money(0) is referenced a few times - might be nice to have a constant for it `Money.ZERO`